### DISCUSSION TITLE
Format of the CSV Import files

### DISCUSSION THREAD

**Discussion by 6illes** [netbox-community/netbox#15463] [Help Wanted!]
Hi, 
we can't find the documentation related to "CSV Import file" via the web interface.
Some posts are talking about REST API , Scripts but I want to use CSV Import.
Thanks

**Comment by jeremystretch** [comment]
You just asked this question [here](https://github.com/netbox-community/netbox/discussions/15461) two hours ago. Please don't make duplicate posts.
